src/js/common/components/Modal.jsx

```diff
@@ -1,10 +1,38 @@
 import React from 'react';
 import classNames from 'classnames';
 
+function focusListener() {
+  const listener = event => {
+    const modal = document.querySelector('.va-modal');
+    if (!modal.contains(event.target)) {
+      event.stopPropagation();
+      modal.focus();
```

Comment:
@U-DON If there were multiple visible, yes, but that wouldn't work with how the modal is displaying right now either. The `va-modal` class only appears to be in the DOM when the modal is visible.